.github/workflows/devbuild.yml

```diff
@@ -200,7 +202,7 @@ jobs:
             VERBOSE=1 USE_CLANG_TIDY=OFF \
             BUILD_QT=OFF \
             CMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }} \
-            CMAKE_ARGS="-DPYTHON_EXECUTABLE=$(which python3)"
+            CMAKE_ARGS="-DPYTHON_EXECUTABLE=$(which python3) -DMODMESH_USE_PYSIDE:BOOL=OFF"
```

Comment:
It is redundant to turn off `-DMODMESH_USE_PYSIDE` when qt is not enabled, but I leave it here for clarity.